spacy/lang/bg/stop_words.py

```diff
@@ -1,265 +1,74 @@
-# Source: https://github.com/Alir3z4/stop-words
```

Comment:
Can you provide a bit more detail why you're removing this URL?

It's difficult to judge from the GH diff, because the list was reformatted, but it looks like you've made some additions & corrections to the original stop list - which means that you didn't start from scratch, correct? If you didn't start from scratch, I think we should keep the note to this link, perhaps rephrase slightly: "Based upon : ..." instead of just "Source".

Reply:
Yes you are right, I forgot about this URL I will add it to the References on top of it with the appropriate explanations. 

I've added corrections & additions to the original one that is correct. I get why it is difficult to see what are the new additions, I wanted to format it like the other languages (ex. EN) so the code would be more coherent.